bubleSort.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$n = $argv[1];
+$randomNumberArray=array();
+$j = 0;
+for ($i = 0; $i < $n; $i++) {
+    $randomNumber = rand();
+    array_push($randomNumberArray, $randomNumber);
+}
+echo "Atsitiktiniu skaiciu masyvas: ";
+var_dump($randomNumberArray);
+echo "\n";
+while ($j< $n) {
```

Comment:
Iškart kyla klausimas, kas tas `$j`, kur jis naudotas anksčiau, kokia gali būti jo reikšmė? Dėl to yra tokia kaip ir nerašyta taisyklė, kad geriausia kitamuosius aprašyti kaip įmanoma vėliau. Šiuo atveju būtų gražu 5 eilutę perkelti iškart prieš while ciklą. Čia FYI kitam kartui.